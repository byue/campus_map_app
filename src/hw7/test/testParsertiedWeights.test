# test parser on a graph with 3 nodes.
# There are two paths with the same weight but one path is of a shorter length.

LoadGraph heroes tiedWeights.tsv

FindPath heroes a c