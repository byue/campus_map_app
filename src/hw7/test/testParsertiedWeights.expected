# test parser on a graph with 3 nodes.
# There are two paths with the same weight but one path is of a shorter length.

loaded graph heroes

path from a to c:
a to c with weight 0.500
total cost: 0.500